docs/ABI.rst

```diff
@@ -1,1230 +0,0 @@
-:orphan:
-
-.. @raise litre.TestsAreMissing
-.. _ABI:
```

Comment:
I think it's fine to remove it. @gribozavr or @dabrahams, is there an equivalent annotation we might use for future structuring of the compiler docs?